### PR TITLE
Chore: Only change operation id for functions with many events

### PR DESCRIPTION
**Pull request by Yossi-kerner**
Only add the event method to the operation id when there are many events that trigger the function to not hurt other users' usage

### CODE REVIEW

File: test/serverless-trigger-events-list/resources/serverless.yml [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+            summary: "Delete Function"
+            description: "Delete"
+          path: update
+          method: put
+      - http:
+          documentation:
+            summary: "Delete Function"
+            description: "Delete"
+          path: update
+          method: post
```
Comment (HeverFarber) [inline test/serverless-trigger-events-list/resources/serverless.yml:84]: why do we need all the file except for this case?

**Review by HeverFarber — COMMENTED** [review]
You should bump hotfix version - and add a light test only for multiple (Don't need all the other cases) or add it to the other tests

File: src/serverless-openapi-typescript.ts [hunk 2]
```diff
@@ -234,7 +235,7 @@ export default class ServerlessOpenapiTypeScript {
                     method.tags = [tagName];
                 }
 
-                method.operationId = kebabCase(`${this.serverless.service.custom?.documentation?.title}-${method.operationId}-${methodName}`);
+                method.operationId = kebabCase(`${this.serverless.service.custom?.documentation?.title}-${method.operationId}${httpEvents.length > 1 ? `-${methodName}`: ''}`);
```
Comment (HeverFarber) [inline src/serverless-openapi-typescript.ts:238]:
```suggestion
                method.operationId = kebabCase(`${this.serverless.service.custom?.documentation?.title}-${method.operationId}${httpEvents?.length > 1 ? `-${methodName}`: ''}`);
```

**Comment by Yossi-kerner** [comment]
not needed anymore

**Pull request closed without merging**